### DISCUSSION TITLE
recipes/breads/cranberry-orange-bread/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1244] [Comments]
# recipes/breads/cranberry-orange-bread/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/cranberry-orange-bread/

**Comment by nicholaswilde** [comment]
- Add orange glaze, same as another lemon blueberry bread. 
- massage zest with sugar.
- Soak cranberries in hot water for .5 hours or overnight.

**Reply by nicholaswilde** [reply]
- Add orange juice to buttermilk as wet ingredient.
- frozen cranberries 
- coat cranberries in flour to prevent sinking.
